data/templates/post-windows-install.ps1

```diff
@@ -0,0 +1,49 @@
+curl http://<%=server%>:<%=port%>/templates/unattend_server2012.xml -Outfile unattend_server2012.xml

```

Comment:
HI @zyoung51, as we talked earlier in the day, I wanted to  keep all the setup information in the xml file.
But I agree with you that it is easier if skip the xml file. 
I fixed it per your suggestion